layouts/partials/data-science.html

```diff
@@ -53,11 +70,11 @@
         </div>
     </div>
     <p>
-        Effective data analytics require deep knowledge of the data domain (e.g.,
-        Retail, Healthcare, Marketing, Finance, Social Media, Automation, Sales, Travel,
-        etc.) as well as other core disciplines of Data Science, Data Engineering and
-        Data Visualization. Tools such as <a href="https://mlflow.org">MLFlow</a> address
-        experiment hyper-parameter and result tracking needs, while
+        Effective data analytics requires deep knowledge of the data domain (e.g.,
+        retail, healthcare, marketing, finance, social media, automation, sales, travel,
+        etc.) as well as other core disciplines of data science, data engineering, and
+        data visualization. Tools such as <a href="https://mlflow.org">MLFlow</a> address
+        experiment hyperparameter and result tracking needs, while
         <a href="https://dvc.org"> DVC</a> provides data version control for data science
         and machine learning workflows.
```

Comment:
Still would like to make this tab a little more compact. Perhaps in a follow-up, all these changes look good.